run-build.sh

```diff
@@ -60,9 +55,6 @@ while [[ $# > 0 ]]; do
     shift
 done
 
-# Set nuget package cache under the repo
-export NUGET_PACKAGES="$REPOROOT/.nuget/packages"
-
 # Set up the environment to be used for building with clang.
```

Comment:
We've just looked through the entire build, I didn't see anywhere that uses clang, or build any native code.